Description: Patch to remove unneeded logtail version detection code in the
 courier plugin
Origin: other, http://bugs.debian.org/298895
Bug-Debian: http://bugs.debian.org/298895
Index: trunk/node/node.d/courier_.in
===================================================================
--- trunk.orig/node/node.d/courier_.in	2008-07-01 19:19:59.000000000 +0200
+++ trunk/node/node.d/courier_.in	2008-07-01 19:20:32.000000000 +0200
@@ -67,25 +67,13 @@
     ;;
 esac
 
-ARGS=0
-`$LOGTAIL /etc/hosts 2>/dev/null >/dev/null`
-if [ $? = 66 ]; then
-    if [ ! -n "$logtail" ]; then
-        ARGS=1
-    fi
-fi
-
 TEMP_FILE=`mktempfile munin-courier.XXXXXX`
 
 if [ -z "$TEMP_FILE" -o ! -f "$TEMP_FILE" ]; then
     exit 3
 fi
 
-if [ $ARGS != 0 ]; then
-    ${LOGTAIL} -f ${COURIER_LOG} -o ${OFFSET_FILE} | grep "$SERVICE" > ${TEMP_FILE}
-else
-    ${LOGTAIL} ${COURIER_LOG} ${OFFSET_FILE} | grep "$SERVICE" > ${TEMP_FILE}
-fi
+${LOGTAIL} ${COURIER_LOG} ${OFFSET_FILE} | grep "$SERVICE" > ${TEMP_FILE}
 connection=`grep Connection ${TEMP_FILE} | wc -l`
 disconnected=`grep DISCONNECTED ${TEMP_FILE} | wc -l`
 login=`grep LOGIN ${TEMP_FILE} | wc -l`
